### PR TITLE
Rel posts

### PR DESCRIPTION
**Pull request by johnedquinn**
2 main changes:
- In /posts/<pid>/related endpoint: grab more info from mews_app.Posts but make sure to only grab info from related post using `IF(post1_id = %(post_id)s, post2_id, post1_id) AS rel_id`.
- In /graph/central: use set to make sure we don't return multiple types of same post. See comment for better explanation.

### CODE REVIEW

**Comment by johnedquinn** [comment]
Also includes the changes from timeConvert branch. See PR #19 for reference.

File: MewsUtils/Posts.py [hunk 1]
```diff
@@ -178,28 +181,47 @@ def getRelatedPosts(pid, skip, amount):
...
+                IF(post1_id = %(post_id)s, post2_id, post1_id) AS rel_id,
+                rel_txt_wt,
+                rel_txt_meta,
+                ocr_meta,
+                sub_img_wt,
+                ocr_wt,
+                ifnull(rel_txt_wt, 0) * %(rel_txt_wt)s + 
+                ifnull(sub_img_wt, 0) * %(sub_img_wt)s + 
+                ifnull(ocr_wt, 0) * %(ocr_wt)s 
+                AS total_wt
```
Comment (danjr26) [inline MewsUtils/Posts.py]: I've recently made total_wt a derived column of the PostRelatedness table, so you can actually just select it directly

**Review by danjr26 — CHANGES_REQUESTED** [review]
Looks good, see my one suggestion

File: MewsUtils/Posts.py (hunk 1 above)
Comment (johnedquinn) [reply MewsUtils/Posts.py]: Just updated. Thanks for heads up.

**Review by danjr26 — APPROVED** [review]

**Pull request merged**